modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/maven/Md5.kt

```diff
@@ -24,7 +24,7 @@ public class Md5 {
          * a checksum of the last modified timestamp.
          */
         fun toMd5Directories(filesOrDirectories: List<File>,
-                toBytes: (File) -> ByteArray = { it.lastModified().toString().toByteArray() } ): String? {
+                toBytes: (File) -> ByteArray = { "${it.path} ${it.lastModified()} ${it.length()}".toByteArray() } ): String? {
```

Comment:
Isn't `length()` superfluous? (doesn't matter much, just curious what you have in mind)